priority-queue.cpp

```diff
@@ -30,7 +30,30 @@ template <typename T>
 void PriorityQueue<T>::removeMin() {
     //TODO: mark
     //remove root
```

Comment:
![image](https://user-images.githubusercontent.com/187711/122898625-9d65cf80-d37d-11eb-8891-2f3502b41f72.png)

Reply:
Remove the first node and with the empty position swap the second node, remove the last node, and apply the down heap bubbling. u is now the current node while v is the child. If v is smaller than u swap the content